src/scripting/game_lua_kernel.cpp

```diff
@@ -103,6 +103,7 @@
 #include "units/map.hpp"  // for unit_map, etc
 #include "units/ptr.hpp"                 // for unit_const_ptr, unit_ptr
 #include "units/types.hpp"    // for unit_type_data, unit_types, etc
+#include "formula/string_utils.hpp"    // for unit_type_data, unit_types, etc
```

Comment:
Well, in this specific case it's pretty clear he copied the preceding line and just changed the filename.